www/js/lib/uiUtil.js

```diff
@@ -227,12 +228,88 @@ define([], function() {
             '/': '&#x2F;',
             '`': '&#x60;',
             '=': '&#x3D;'
-          };
-          string = String(string).replace(/[&<>"'`=/]/g, function (s) {
-            return escapechars[s];});
-          return string;
+        };
+        string = String(string).replace(/[&<>"'`=/]/g, function (s) {
+            return escapechars[s];
+        });
+        return string;
+    }
 
+    /**
+     * Removes the animation effect between various sections
+     */
+    function removeAnimationClasses() {
+        $('#about').removeClass('slideIn_L').removeClass('slideOut_R');
+        $('#configuration').removeClass('slideIn_L').removeClass('slideIn_R').removeClass('slideOut_L').removeClass('slideOut_R');
+        $('#articleContent').removeClass('slideIn_R').removeClass('slideOut_L');
+    }
+    
+    /**
+     * Adds the slide animation between different sections
+     * 
+     * @param {String} name_of_section It takes the name of the section to which the animation is to be added
```

Comment:
The param name is wrong here

Reply:
Should it be 'string' instead of 'name_of_section' 
Something like -
@param {String} string It takes the name of the section as string to which the animation is to be added

Reply:
@vasusharma7 It should be `@param {String} section The name of the section to which the animation is to be added`

Reply:
Thanks @Jaifroid 
I was  a bit confused